### PR TITLE
[MOSIP-28624]Update pms-migration-utility-default.properties

### PR DESCRIPTION
**Pull request by balaji-alluru**

### CODE REVIEW

File: pms-migration-utility-default.properties [hunk 1]
```diff
@@ -64,7 +68,7 @@ hibernate.generate_statistics=false
 spring.datasource.initialization-mode=never
 
 # websub related properties
-websub.publish.url=mosip.websub.url
+websub.publish.url=${mosip.api.internal.url}/hub/
```
Comment (vishwa-vyom) [inline pms-migration-utility-default.properties]: why do we use the external URL here why not internal URL ?

File: pms-migration-utility-default.properties [hunk 2]
```diff
@@ -16,9 +16,13 @@ mosip.kernel.authmanager.url=http://authmanager.kernel
 
 spring.main.allow-circular-references=true
 service.token.request.clientId=mosip-pms-client
-service.token.request.issuerUrl=${mosip.api.internal.url}/v1/authmanager/authenticate/clientidsecretkey
+service.token.request.issuerUrl=${mosip.kernel.authmanager.url}/v1/authmanager/authenticate/clientidsecretkey
```
Comment (vishwa-vyom) [inline pms-migration-utility-default.properties:19]: Please update the JIRA id in the PR

**Review by vishwa-vyom — CHANGES_REQUESTED** [review]

File: pms-migration-utility-default.properties (hunk 1 above)
Comment (balaji-alluru) [reply pms-migration-utility-default.properties]: with that websub url, we are getting an error. Rakshit also reported the same sir. That is why I updated that property

File: pms-migration-utility-default.properties (hunk 2 above)
Comment (balaji-alluru) [reply pms-migration-utility-default.properties:19]: added the same

File: pms-migration-utility-default.properties (hunk 1 above)
Comment (vishwa-vyom) [reply pms-migration-utility-default.properties]: Can you try with this ?
${websub.publish.url}

Comment (balaji-alluru) [reply pms-migration-utility-default.properties]: we tried with that sir, we used the place holder, still we got the same error.

Comment (vishwa-vyom) [reply pms-migration-utility-default.properties]: Can you share the exact error here ?

Comment (balaji-alluru) [reply pms-migration-utility-default.properties]: Here is the exception that we are getting with that websub url

`io.mosip.kernel.websub.api.exception.WebSubClientException: KER-WSC-103 --> Error occured while publishing topic to hub :- no matching service found for path : /
	at io.mosip.kernel.websub.api.client.PublisherClientImpl.publishUpdate(PublisherClientImpl.java:133)
	at io.mosip.kernel.websub.api.client.PublisherClientImpl.publishUpdate(PublisherClientImpl.java:35)
	at io.mosip.pms.ida.websub.WebSubPublisher.sendEventToIDA(WebSubPublisher.java:44)
	at io.mosip.pms.ida.websub.WebSubPublisher.notify(WebSubPublisher.java:32)
`

Comment (balaji-alluru) [reply pms-migration-utility-default.properties]: updated the websub property with the internal URL as suggested sir.

**Review by sowmya695 — APPROVED** [review]

**Review by ckm007 — APPROVED** [review]

File: pms-migration-utility-default.properties [hunk 3]
```diff
@@ -64,7 +68,8 @@ hibernate.generate_statistics=false
 spring.datasource.initialization-mode=never
 
 # websub related properties
-websub.publish.url=mosip.websub.url
+mosip.websub.url=http://websub.websub
```
Comment (vishwa-vyom) [inline pms-migration-utility-default.properties:71]: I think this property is already there in application-default.properties, why do we need this again here ?

**Review by vishwa-vyom — CHANGES_REQUESTED** [review]

**Review by vishwa-vyom — APPROVED** [review]

**Pull request merged**